release.js

```diff
@@ -14,15 +13,20 @@ const writeFile = pify(fs.writeFile)
 const dryRun = process.argv.includes('--dry-run')
 const {version} = require('./package.json')
 
+let semver
+['patch', 'minor', 'major'].forEach(i => { process.argv.includes(i) && (semver = i) })
+assert.ok(semver, 'Must specify the valid semver version: patch | minor | major')
+
 ;(async () => {
   log('Doing sanity checks...')
   const {currentBranch: branchToPublish, isCleanWorkingTree} = await getGitStatus()
   if (!dryRun) {
     assert.equal(branchToPublish, 'master', 'Must be on master branch')
   }
   assert.equal(isCleanWorkingTree, true, 'Must have clean working tree')
-  const gitTags = await getGitTags()
-  assert.ok(!gitTags.includes(`v${version}`), 'Must have a unique version in package.json')
+
+  log(`Running npm version ${semver}...`)
+  await execute(`npm version --no-git-tag-version ${semver}`)
```

Comment:
Maybe under this line we can add:
```js
const {version} = require('./package.json')
await execute (`git add package.json && git commit -m "v${version}"`)
```
Addresses https://github.com/sweetalert2/sweetalert2/pull/880#issuecomment-362711405 and https://github.com/sweetalert2/sweetalert2/pull/880#issuecomment-362713538

Then I thing a more appropriate log message would be something like "Bumping package version...`